jobs/birthdayOfTheDay.ts

```diff
@@ -0,0 +1,40 @@
+import { TextChannel } from "discord.js"
+import moment from "moment";
+
+async function getBdayUsers(date) 
+{
+  const users = await prisma.user.findMany({
+    where: {
+        birthday: {
+          birthday: date
+        }
+    }
+  })
+  
+  const arrayIds = []
+  
+  for (let i = 0; i < users.length; i++)
```

Comment:
instead of doing a computation for just putting in the ids in an array, just return the whole users array from the db query